mgmt/rest/v2/response/metric.go

```diff
@@ -91,3 +84,8 @@ func (s StreamedMetrics) Less(i, j int) bool {
 func (s StreamedMetrics) Swap(i, j int) {
 	s[i], s[j] = s[j], s[i]
 }
+
+// cdata.ConfigDataNode implements it's own UnmarshalJSON
```

Comment:
Yes. Actually, I did add the comment because unmarshalling `ConfigDataNode` is customized.